caravel/assets/javascripts/SqlLab/common.js

```diff
@@ -5,4 +5,18 @@ export const STATE_BSSTYLE_MAP = {
   success: 'success',
 };
 
-export const STATUS_OPTIONS = ['success', 'failed', 'running'];
+export const STATUS_OPTIONS = [
```

Comment:
seems like this file should be called `constants` rather than `common` for clarity.